packages/typedoc-plugins/src/purge-private-api-docs.ts

```
@@ -0,0 +1,200 @@
+    });
+};
+
+/**
+ * Removes URL sources from the given reflection recursively.
+ *
+ * @param reflection - The reflection to process.
+ */
+const removeUrlSourcesFromReflection = (reflection: DeclarationReflection): void => {
+    if (reflection.sources) {
+        reflection.sources.forEach((source) => {
+            delete source.url;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)